locales/en-US/app.l20n

```diff
@@ -73,6 +74,17 @@
 <feedbackUninstallAnswerNotUseful "This isn't useful for me.">
 <feedbackUninstallAnswerOther "Something else.">
 
+<retireDialogTitle "Retire from Test Pilot?">
+<retireMessage "If you retire from Test Pilot, we will disable any active tests, uninstall the Test Pilot add-on, remove your account information from our servers, and generally just leave you alone.">
+<retireSubmitButton "Retire">
+<retireCancelButton "cancel">
```

Comment:
Lowercase or uppercase "c"?
I see on L69 above, we have an uppercase "Cancel" for the `feedbackCancelButton` label.